### ISSUE TITLE
Feature request: Backup/Export accounts

### ISSUE THREAD

**Issue by ztl8702** [YC/another_authenticator#1]
Backup and/or export/import accounts.

I can work on it if I have time.

**Comment by YC** [comment]
May I ask how you plan to implement it? 
I'm a bit concerned about security. 
I think at least a log of some sort will need to be updated when users view or transfer accounts. 

Also, I'm happy to transfer the repo to an organisation if you want to work on it.

**Comment by ztl8702** [comment]
Do you currently store the keys as plaintext?

I am thinking storing them as an encrypted file that requires PIN on app launch to decrypt. So the exported file will also be encrypted and requires PIN when importing.

**Comment by YC** [comment]
Yes, it is currently a JSON file in plaintext. 
If a PIN is required, then I guess that has to be implemented as a part of the app itself?

**Comment by ztl8702** [comment]
> If a PIN is required, then I guess that has to be implemented as a part of the app itself?

Yeah

**Comment by bhargavraviya** [comment]
Yes, we need export features. and convert to framework like Getx ?

@YC
